packages/fileeditor/src/widget.ts

```diff
@@ -29,11 +33,8 @@ const DIRTY_CLASS = 'jp-mod-dirty';
 const EDITOR_CLASS = 'jp-FileEditor';
 
 
-/**
- * A document widget for editors.
- */
 export
-class FileEditor extends CodeEditorWrapper implements DocumentRegistry.IReadyWidget {
+class FileEditorInner extends CodeEditorWrapper implements DocumentRegistry.IReadyWidget {
```

Comment:
I'd say call this `FileEditorCodeWrapper` and not have it implement `DocumentRegistry.IReadyWidget` explicitly because it is not intended to be used as one.